TedOnyango/Politico
# Implementing liting and pep8 on my code

**What does this PR do?**
Implements feedback on linting and use of pep8

**Description of Task to be completed?**
Liting the code with pylint
Implementing pep8 standards 

**How should this be manually tested?**
 After cloning the repository to your local machine - check README.md for instructions. Using Postman open create a request with this header Key Content-Type: application/json
